test/src/services/institute-api-service.js

```diff
@@ -38,6 +45,8 @@ const instituteApiService = {
     };
     const url = `${constants.institute_base_url}${SCHOOL_ENDPOINT}/paginated`;
     const userSchoolResult = await restUtils.getData(token, url, schoolSearchParam);
+    console.log('SCHOOL_RESULT:== ');
+    console.log(userSchoolResult);
```

Comment:
I'm not sure if you meant to use console.log here. Usually when we want to log something, we use log.info in tests. 

Could also use a log.debug if you dont' always want to show the school results.